schemas/context/experienceevent.example.1.json

```diff
@@ -114,5 +114,48 @@
   },
   "xdm:marketing": {
     "xdm:trackingCode": "marketingcampaign111"
-  }
+  },
+  "xdm:profileStitching": [
```

Comment:
A couple of questions:
- What is the use case for adding profileStitching to ExperienceEvent? 
- Is the user expected to populate this via ETL jobs?

Reply:
To support profile stitching in realtime for use with realtime segmentation
Someone could populate this with an ETL job if they wanted, but that wasn't the use case I was considering. I think it's open enough to support that.